torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1564,20 +1588,20 @@ def sample_inputs_take_along_dim(op_info, device, dtype, requires_grad, **kwargs
             )
 
 def sample_inputs_amax_amin(op_info, device, dtype, requires_grad, **kwargs):
-    test_cases = (
-        ((S, S, S), ()),
-        ((S, S, S), (1,)),
-        ((S, S, S), ((1, 2,),)),
-        ((S, S, S), (1, True,)),
-        ((), (0,)),
-        ((), ()),
-        ((), (0, True,)),
+    test_cases: Tuple[tuple, tuple, dict] = (  # type: ignore[assignment]
+        ((S, S, S), (), {}),
+        ((S, S, S), (1,), {}),
+        ((S, S, S), ((1, 2,),), {}),
+        ((S, S, S), (1,), {'keepdim': True}),
+        ((), (0,), {}),
+        ((), (), {}),
+        ((), (0,), {'keepdim': True}),
```

Comment:
**Why this change?**

Example function: `np.amax` (https://numpy.org/doc/stable/reference/generated/numpy.amax.html)

- NumPy takes `out` as a keyword argument while PyTorch takes `out` as kw-only argument.
- NumPy function's signature has `out` as the 3rd argument (/or an argument which doesn't match the position of `keepdim` from `sample_inputs`).

For example, here, if the earlier case is considered, and we call the reference function: `ref_fn(inp, *args, **kwargs)` (where kwargs is `{}`), then boolean is going to be passed to `out=` param in NumPy's amax function call. Which is incorrect and will throw an error.

(continuing from the points above)

- OpInfo already has ability to handle `kwargs` from sample inputs functions.
- Since `keepdim` is a keyword argument in PyTorch, so it does make sense to pass as a kwarg.
- This will resolve the above error of function call assuming `keepdim` values passed as `out=` values.

(Tested OK)

**Notes**:

- For future PRs on adding reference functions, this change will be desired if there is `keepdim` like argument.